packages/client/src/context/Auth.context.tsx

```diff
@@ -6,6 +6,13 @@ import * as firebaseauth from '@firebase/auth';
 import { Organization } from '../graphql/graphql';
 import { useGetOrganizationsQuery } from '../graphql/organization/organization';
 
+import styles from './UIComponents/styles.component';
+import LoginComponent from './UIComponents/Login.component';
```

Comment:
All the components should be stored in the `packages/client/src/components` folder. You can make a subfolder within there for the auth components if you like